### PR TITLE
Fixes #1964 Adding notice to wallet about Stacks Wallet

### LINKED ISSUE #1964: Add notice to wallet route distinguishing Stacks Wallet
Currently, visitors to [the Blockstack Browser's "Wallet" route](https://browser.blockstack.org/wallet/receive) are prone to confuse the Browser's BTC wallet shown there with [the separate Stacks Wallet](https://wallet.blockstack.org/) for STX tokens.

This issue entails adding a notice to the top of this route's view that alerts the user to the distinction and directs all users interested in the Stacks Wallet to its website.

### PR DESCRIPTION
**Pull request by moxiegirl**
This PR implements the following:

image

Signed-off-by: Mary Anthony <mary@blockstack.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
Changed text to this:

image

**Comment by moxiegirl** [comment]
@hstove assigned you to this for the merge/build step

**Comment by hstove** [comment]
This is great! Thanks, @moxiegirl

File: app/js/wallet/WalletApp.js [hunk 1]
```diff
@@ -32,23 +32,28 @@ class WalletApp extends Component {
...
-          customButtonClass="btn-wallet"
-        />
-        <div className="container-fluid col-centered form-container-secondary">
-          <div>
-            {this.props.children}
+      <div className="container-fluid">
+        <div>
+          <Navbar activeTab="wallet" />
+          <div className="container-fluid col-centered form-container-secondary">
+            <strong>NOTE:</strong> You cannot use this wallet to send and receive Stacks (STX) tokens. Also, you cannot use the Bitcoin (BTC) address on this page to fund STX transactions. This wallet and its address <strong>only</strong> support the purchase of Blockstack identities(IDs). <strong>To create or fund STX transactions, use the Stacks Wallet software.</strong> See <a href="https://docs.blockstack.org/org/wallet-install.html" target="_blank">the Stacks Wallet software documentation</a> for more information.
```
Comment (markmhx) [inline app/js/wallet/WalletApp.js]: This PR has been merged but I spot the need for a space here before "(IDs)" so @moxiegirl or @hstove you may want to create a small new PR to correct.

**Pull request merged**